roles/sushy_emulator/tasks/apply_resources.yml

```diff
@@ -0,0 +1,41 @@
+- name: Create the sushy-emulator namespace
```

Comment:
nit, but for consistency, this is missing the yaml header (not sure if that's the right term I mean the `---`) and the copyright notice